### PR TITLE
docker: Move the docker document and compose to a separate dir.

### PR DESCRIPTION
**Pull request by harshavardhana**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/docker/README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ docker run -p 9000:9000 --name minio1 \
 
 ## 4. Run Distributed Minio on Docker
```
Comment (abperiasamy) [inline docs/docker/README.md]: Test Distributed..

**Review by abperiasamy — COMMENTED** [review]
Compose is only meant for testing (simulating) a distributed Minio setup on a single node. It is not meant for real workload. In that case, it is OK to simply use internal ~/.minio and /export

**Comment by neomantra** [comment]
The `docker-compose.yml` link on [this page](https://docs.minio.io/docs/docker-compose.yml) is currently broken.  Maybe it needs to be regenerated due to this commit?

**Comment by harshavardhana** [comment]
> The docker-compose.yml link on this page is currently broken. Maybe it needs to be regenerated due to this commit?

No looks like the path is wrong. Will fix it thanks for reporting @neomantra it is accessible here - https://github.com/minio/minio/blob/master/docs/docker/docker-compose.yml

**Pull request merged**